test/cluster/cluster.go

```diff
@@ -500,6 +502,19 @@ func lookupUser(name string) (int, int, error) {
 }
 
 func (c *Cluster) DumpLogs(w io.Writer) {
+	done := make(chan struct{})
+	go func() {
+		c.dumpLogs(w)
+		close(done)
+	}()
+	select {
+	case <-done:
+	case <-time.After(60 * time.Second):
```

Comment:
This timeout covers all the log gather attempts at once... do we need the big one, if the timeout per ssh connection is now low?